web/crux/src/app/version/version.service.ts

```diff
@@ -118,9 +118,16 @@ export default class VersionService {
                 versionId: newVersion.id,
                 config: {
                   create: {
-                    ...this.imageMapper.configDetailsToDb(image.config),
+                    ...image.config,
+                    id: undefined,
                   },
                 },
+                // TODO TEST FOR MAPPING
```

Comment:
You will create tests or what'll be the action item? 
(Add your Github name please to Todo comment)

Reply:
i will delete the todos in this pr after manual testing

Reply:
Okey, cool.